### DISCUSSION TITLE
Data importer: how do you specify name of created objects?

### DISCUSSION THREAD

**Discussion by FredJowett** [pimcore/pimcore#12053] [Q&A / Support] [answered]
Hi,

I'm trying to import data objects from XLSX, and while my importer configuration correctly executes, it creates new objects with a name of `import-<digits>`.  Is it possible to specify the name for the created object, ideally from an attribute in the imported data?  I couldn't see anything regarding this in the documentation.

Cheers,
Fred

**Comment by kingjia90** [comment, marked as answer]
You can do that by mapping the Data Target to `SYSTEM KEY` with any source attributes you wish/need (eg. in the screenshot `manufacturer` and `model`)
![image](https://user-images.githubusercontent.com/6014195/167146720-33681d48-95fb-4946-b725-a61017bc3a1a.png)

**Reply by FredJowett** [reply]
Yes, that worked!  Thanks very much!
